### ISSUE TITLE
Are published snap packages official releases?

### ISSUE THREAD

**Issue by RogueScholar** [bitpay/wallet#9288]
I see that recently packages were added to Snapcraft.io for both the [BitPay](https://snapcraft.io/bitpay) and [CoPay](https://snapcraft.io/copay) wallets. Are these official releases?

[Last I heard](https://github.com/bitpay/copay/issues/8767) the decision had been made not to package the wallets in either deb or snap formats. I'm no fan of snaps in general, but if these are legit I was going to audition the BitPay snap to see how well it automates keeping the wallet software updated.

**Comment by BGalliotti** [comment]
Hello @RogueScholar 
Thanks for reaching out to us.

Our developers will be answering your inquiry soon, we would appreciate your patience.

Thanks in advance.

Please let us know if you have any additional questions.
Kind regards.

**Comment by gustawdaniel** [comment]
I am too interested in obtaining official confirmation from your team. Initially I was totally surprised and happy that it was released, but no any links with my issue mentioned by @RogueScholar make me careful.

https://snapcraft.io/search?category=&q=bitpay

**Comment by gabrielbazan7** [comment]
Yes, those are official releases. The GitHub versions are no longer maintained and if you use to have this desktop version you will need to migrate your wallets manually to the new version

**Comment by gustawdaniel** [comment]
Thank You very much. I love this decision and you are awesome!

**Comment by RogueScholar** [comment]
Thanks for clearing everything up. I've done as you suggested and migrated my wallets to the snap app successfully.

The only issue I've found with it vs. the old binary release is the inability to link my BitPay VISA via either the link on the website or from an e-mail. I've confirmed that xdg-mime does show the snap's .desktop file as the default for x-scheme-handler/bitpay.

Could it be that the lack of a %U at the end of the .desktop file's Exec line is the cause?
